### PR TITLE
Add publishing for common

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: common/build.gradle.kts [hunk 1]
```diff
@@ -1,8 +1,12 @@
-plugins {
-    java
-}
+group = "org.jboss.gm.common"
 
-version = "0.1-SNAPSHOT"
+publishing {
+    publications {
+        register("mavenJava", MavenPublication::class) {
```
Comment (metacosm) [inline common/build.gradle.kts]: Why is this needed?

Comment (rnc) [reply common/build.gradle.kts]: Trying to get the common module published so that the plugins can use it
